src/vivarium_cluster_tools/vadmin/log_parser.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, file: Path):
         self.task_id = int(name[1])
 
     def _load_messages(self):
-        with open(self.file) as f:
+        with self.file.open('r') as f:
```

Comment:
is this going to be our preferred method going forward ?

Reply:
This is the preferred method when you're hanging onto a pathlib.Path object.  The whole point of the library is object oriented file utilities.